assembl/graphql/schema.py

```diff
@@ -46,6 +54,17 @@ def get_node(cls, id, context, info):
 # manually.
 
 
+def get_entries(langstring):
+    # langstring.entries is a backref, it doesn't get updated until the commit.
+    # Even with db.flush(), new entries doesn't show up or a specific entry show up
```

Comment:
I know there is a expire_on_flush=True option for a column_property. But I didn't see an equivalent for a backref.